tempto-core/src/main/java/io/prestosql/tempto/internal/listeners/TestNameGroupNameMethodSelector.java

```diff
@@ -54,9 +55,9 @@
     public static final String TEST_GROUPS_TO_RUN_PROPERTY = "io.prestosql.tempto.groups";
     public static final String TEST_GROUPS_TO_EXCLUDE_PROPERTY = "io.prestosql.tempto.exclude_groups";
 
-    private final Optional<Set<String>> testNamesToRun;
-    private final Optional<Set<String>> testGroupsToRun;
-    private final Optional<Set<String>> testGroupsToExclude;
+    private final Set<String> testNamesToRun;
```

Comment:
This is a semantics change for `testNamesToRun`.
-- previously, Optional.empty() meant: no limit and Optional.of(Set.of()) meant: run nothing.

In my comment i meant `testGroupsToExclude` only.

Also, i think `testNamesToRun = Optional.empty()` (the old way) is a better way of saying "not specified" than `testNamesToRun = Set.of()` (the new way)

Reply:
Oh, you are right.  I didn't read that carefully.